Makefile

```diff
@@ -1,4 +1,4 @@
-SITE=https://www.perl.com
+SITE=https://perldotcom.perl.org
```

Comment:
We don't need to change this one just yet. This simply supports a tool that I personally use and I already have other measures in place.